tests/test_lib.py

```diff
@@ -216,11 +217,7 @@ class TestEnum(Enum):
     } == convert(vol.Schema(vol.Coerce(TestEnum)))
 
 
-def test_invalid_schema():
-    # check if an exception is raised when an invalid schema is passed
-    try:
-        convert(vol.Schema(None))
-    except Exception as e:
-        assert str(e) == "Unable to convert schema: None"
-    else:
-        assert False, "Expected an exception to be raised"
+@pytest.mark.parametrize("invalid_schema", [None, []])
+def test_invalid_schema(invalid_schema):
+    with pytest.raises(ValueError):
+        convert(vol.Schema(invalid_schema))
```

Comment:
```suggestion
def test_raise_unhashable():
     with pytest.raises(ValueError, match=r"^Unable to convert schema:"):
         convert(vol.Schema([]))
```
with the latest merge of #152 this would fit better to the already existing tests.

Reply:
Let's follow up with the test adjustments. :+1: